bundles/org.openhab.binding.androidtv/src/main/java/org/openhab/binding/androidtv/internal/protocol/philipstv/PhilipsTVConnectionManager.java

```diff
@@ -230,9 +231,9 @@ private void readConfigs() {
             Map<String, String> configMap = OBJECT_MAPPER.readValue(configJson,
                     new TypeReference<HashMap<String, String>>() {
                     });
-            this.username = Optional.ofNullable(configMap.get("username")).orElse("");
-            this.password = Optional.ofNullable(configMap.get("password")).orElse("");
-            this.macAddress = Optional.ofNullable(configMap.get("macAddress")).orElse("");
+            this.username = Objects.requireNonNullElse(configMap.get("username"),"");
```

Comment:
I think you still need to run `spotless:apply`:

```suggestion
            this.username = Objects.requireNonNullElse(configMap.get("username"), "");
```

Reply:
Fixed